src/canmatrix/canmatrix.py

```diff
@@ -1085,14 +1085,15 @@ def safeConvertStrToInt(inStr):
 
         elif definition[0:4] == 'ENUM':
             self.type = 'ENUM'
-            tempValues = definition[5:].split(',')
+            tempValues = canmatrix.utils.my_comma_splitter(definition[5:])
             self.values = []
             for value in tempValues:
-                value = value.strip()
-                if value[0] == '"':
-                    value = value[1:]
-                if value[-1] == '"':
-                    value = value[:-1]
+                value = value.replace("vector_leerstring", " ")
```

Comment:
Should `"vector_leerstring"` be here?  What is it?

And drop the commented code below.

Reply:
`vector_leerstring` is candb++ way to set an empty string (found out by @blem14 in #263)
 [`leerstring` is german for `empty string`]

Reply:
as written above - this is heavy dbc driven. 
Defines should not parse strings in canmatrix, should be done in dbc.  
In another PR I'm gonna cut this out and merge it in  dbc module.

Reply:
Didn't they say it was `""` not `" "`?

Reply:
I'm not sure, will check this tomorrow

Reply:
I am sorry you had to wait for response so long, `vector_leerstring` is representing empty string `""`.
The CANdb++ recognizes the `""` when opening a .dbc file, but upon saving it is replaced with `"vector_leerstring"`.